db/bolt/template.go

```diff
@@ -78,6 +78,12 @@ func (d *BoltDb) GetTemplates(projectID int, filter db.TemplateFilter, params db
 			return
 		}
 
+		templates[i].Vaults, err = d.GetTemplateVaults(projectID, templates[i].ID)
```

Comment:
What reason to get vaults here? It creates a lot of requests to database.

As I understand the vault model required only for creating task.

Reply:
I believe it's needed here so that elsewhere in the code the `Vaults` are available. Without this, the Backup `json` file is missing the Vaults, as well as the Get Templates API endpoint.

Reply:
One second, will check. Thank you

Reply:
Yes, you are right. I was sure I added this to the backups.

I will fix vault backups.

Reply:
Done.